### PR TITLE
module to handle constants added; package for mflike created

### PR DESCRIPTION
**Pull request by mgerbino**
This PR addresses Issue #34:

the module constants.py is created in soliket to include any constants needed by other packages and/or modules. It is currently imported by theoryforge_MFLike and clusters. It imports the scipy.constants module and is complemented with other custom-made variables that are defined as constants (e.g., T_CMB).

In addition, the package mflike has been created to mirror other likelihood packages (e.g., lensing, clusters, xcorr).

### CODE REVIEW

File: soliket/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from .constants import *
```
Comment (timothydmorton) [inline soliket/__init__.py:1]: I'd prefer not to have the constants live at the top package level.  Not a strong preference, but I think it's fine to explicitly import the constants when needed.

File: soliket/clusters/clusters.py [hunk 2]
```diff
@@ -9,11 +9,12 @@
 import pyccl as ccl
 
 from ..poisson import PoissonLikelihood
+from ..constants import C_KM_S
 from . import massfunc as mf
 from .survey import SurveyData
 from .sz_utils import szutils
 
-C_KM_S = 2.99792e5
+#C_KM_S = 2.99792e5
```
Comment (timothydmorton) [inline soliket/clusters/clusters.py:17]: delete instead of comment out

File: soliket/constants.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import numpy as np
```
Comment (timothydmorton) [inline soliket/constants.py:1]: remove unnecessary import

File: soliket/mflike/theoryforge_MFLike.py [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+
 def _cmb2bb(nu):
+
     # NB: numerical factors not included
-    from scipy import constants
-    T_CMB = 2.72548
-    x = nu * constants.h * 1e9 / constants.k / T_CMB
+    #from scipy import constants
+    #T_CMB = 2.72548
+    #x = nu * constants.h * 1e9 / constants.k / T_CMB
```
Comment (timothydmorton) [inline soliket/mflike/theoryforge_MFLike.py:15]: remove these lines

Comment (mgerbino) [reply soliket/mflike/theoryforge_MFLike.py]: done!

File: soliket/clusters/clusters.py (hunk 2 above)
Comment (mgerbino) [reply soliket/clusters/clusters.py]: done!

**Pull request merged**